src/components/ResourceCard/AddResource/ResourceDescribeActivity.js

```diff
@@ -1,69 +1,50 @@
-/* eslint-disable max-len */
 import React, { useState, useRef } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router-dom';
 import { Field, reduxForm } from 'redux-form';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-
+import Swal from 'sweetalert2';
 import { required, FadeDiv } from 'utils';
+
```

Comment:
Remove this line and add one above the utils import

Reply:
fixed